server.js

```diff
@@ -28,20 +48,23 @@ const fastify = Fastify({
   logger: true,
 });
 
+Sentry.setupFastifyErrorHandler(fastify);
 await fastify.register(envPlugin);
 
-fastify.register(
-  helmet,
-  {
-    contentSecurityPolicy: {
-      directives: {
-        "connect-src": ["'self'", "sdk.openui5.org"],
-        "script-src": isLocalDev ? ["'self'", "'unsafe-inline'"] : ["'self'"],
-        "frame-ancestors": [fastify.config.FRAME_ANCESTORS]
-      },
-    }
-  }
-)
+let sentryHost = null;
+if (fastify.config.BFF_SENTRY_DSN && fastify.config.BFF_SENTRY_DSN.length > 0) {
```

Comment:
I think this needs to be `VITE_SENTRY_DSN` and not for the backend.
Could also wrap the host extraction it in `try`/`catch` to make it more robust

Reply:
We need two DSN, this is why we have VITE_SENTRY_DSN and BFF_SENTRY_DSN.
I will add TryCatch to the URL parsing